src/custom/CatalogCard/CatalogCard.tsx

```diff
@@ -38,8 +38,21 @@ type CatalogCardProps = {
   cardWidth: string;
   cardStyles: React.CSSProperties;
   type: string;
+  version?: string;
 };
 
+export const VersionTag = styled('div')(({ theme }) => ({
+  position: 'absolute',
+  bottom: '65px',
+  left: '18px',
+  backgroundColor: theme.palette.grey[500],
```

Comment:
not these, there is tokens we use you will get those in palette.js file.

Reply:
> not these, there is tokens we use you will get those in palette.js file.

Thank you for sharing this, I'll make sure to take a look at it and modify this.